docs/source/usage.rst

```diff
@@ -0,0 +1,192 @@
+Usage
+=====
+
+.. contents::
+
+Most common options
+-------------------
+
+The following exemple uses all most common options. 
+It will add a link to your project website in all pages header and show a link to source code aside every documented elements. 
+
+::
+
+    pydoctor \
+        --add-package=(packagedirectory) \
+        --project-name=(name) \
+        --project-url=https://github.com/(user)/(repo)/ \
+        --html-viewsource-base=https://github.com/(user)/(repo)/tree/(tag_or_commit_sha) \
+        --make-html \
+        --html-output=apidocs \
+        --project-base-dir="$(pwd)" \
+        --docformat=restructuredtext 
```

Comment:
Maybe update this example with the actual pydoctor call on the pydoctor source code.

For example, someone should be able to clone the pydoctor repo, run `pip install -e .` and then copy paste the command from here and see the results.

Maybe also explain that the output is find in the `apidocs` directory.

I would prefer to have the ouput in `build/apidocs` to make it clear the the `apidocs` files are generated and not part of the repo.